### DISCUSSION TITLE
Change reporting frequency? (Sonoff plug)

### DISCUSSION THREAD

**Discussion by PaulHamdinger** [Koenkk/zigbee2mqtt#15183] [Devices]
I get an "attributeReport" message in my log file every 10 seconds, from each of the eight Sonoff plugs (S40ZBTPB) I have. It seems to be reporting on its on/off state. This gets pretty noisy. I don't think (?) that these are responses to any messages originating from Z2M ; the messages from the plugs seem to arrive unrequested.

I can't figure out how to change this frequency, if it's even possible. Any suggestions? I'd be fine with maybe, say, one report every 10 minutes.

```
debug 2022-11-25 12:34:16 Received Zigbee message from 'sonoff-plug-08', type 'attributeReport', cluster 'genOnOff', data '{"onOff":0}' from endpoint 1 with groupID 0
info 2022-11-25 12:34:16 MQTT publish: topic 'zigbee2mqtt/sonoff-plug-08', payload '{"last_seen":"2022-11-25T18:34:16.100Z","linkquality":51,"state":"OFF"}'
```

**Comment by TheNutHouse84** [comment]
Did you ever find a solution? Mine do the same and generate unnecessary traffic on the zigbee network.

Comparison between the Sonoff (1500 messages), ThirdReality (60 messages) over approximately 8 hours.

**Reply by PaulHamdinger** [reply]
Sorry, I never did come across (or invent) a solution. For me, the Sonoff plugs have been the most unreliable part of my setup, so I'll probably replace them eventually. Not a great solution to the underlying noisy-network problem, but oh well. :D
